src/class-wpml-gutenberg-strings-registration.php

```diff
@@ -125,6 +125,55 @@ private function register_blocks( array $blocks, array $package_data, $post_id )
 		}
 	}
 
+	/**
+	 * @param array $blocks
+	 * @param array $package_data
+	 */
+	public function register_blocks_from_widget( array $blocks, array $package_data ) {
```

Comment:
Except the filters, it seems to be the same code as in `WPML_Gutenberg_Strings_Registration::register_blocks`. Did you try to call directly `WPML_Gutenberg_Strings_Registration::register_blocks` with `$post_id` equal to `0` or `null`?

Reply:
@gordonsh24